docs/graphql/manual/queries/custom-functions.rst

```diff
@@ -395,6 +395,51 @@ Search nearby landmarks with ``distance_kms`` default value which is 2 kms:
     }
 
 
+Accessing Hasura session variables in custom functions
+******************************************************
+
+Use the v2 :ref:`track_function <track_function_v2>` to add a function by defining a session argument.
+The session argument will be a JSON object where keys are session variable names (in lower case) and values are strings.
+Use the ``->>`` JSON operator to fetch the value of a session variable as shown in the following example.
+
+.. note::
```

Comment:
Can we have the note in the end of this section? When we say `in the following example`, then the example should follow immediately.